Makefile

```diff
@@ -26,7 +26,7 @@ install:
 
 vendor-update:
 	@echo Updating vendored packages
-	@GO15VENDOREXPERIMENT=1 glide -q up
+	@GO15VENDOREXPERIMENT=1 glide install
```

Comment:
If the package is already installed, does option 'install' updates it? What's the reason of removing 'up' ?